src/cdomains/valueDomain.ml

```diff
@@ -2,10 +2,7 @@ open Cil
 open Pretty
 open GobConfig
 
-module ID = IntDomain.IntDomTuple
-module IndexDomain = IntDomain.IntDomWithDefaultIkind (ID) (IntDomain.PtrDiffIkind)
-module AD = AddressDomain.AddressSet (IndexDomain)
-module Addr = Lval.NormalLat (IndexDomain)
+include PreValueDomain
```

Comment:
Why did we move this?

Reply:
I wanted queries to refer to the same `Addr` without having to repeat all the preliminary definitions. If not extracted out to a third module, there's a circular dependency `ValueDomain -> Queries -> ValueDomain`.